crates/prune/src/pruner.rs

```diff
@@ -225,7 +233,13 @@ impl<DB: Database> Pruner<DB> {
         let elapsed = start.elapsed();
         self.metrics.duration_seconds.record(elapsed);
 
-        trace!(target: "pruner", %tip_block_number, ?elapsed, "Pruner finished");
+        info!(
+            target: "pruner",
+            %tip_block_number,
+            %done,
+            elapsed_nanos = elapsed.subsec_nanos(),
+            %total_pruned_entries
+        );
```

Comment:
it misses the log message, and I think logging the duration using `Debug` implementation would be better: https://github.com/rust-lang/rust/blob/e7f9f48d767c1e17e125d83e3102d568f87b868d/library/core/src/time.rs#L1187-L1209

```suggestion
        info!(
            target: "pruner",
            %tip_block_number,
            %done,
            ?elapsed,
            %total_pruned_entries,
            "Pruner finished"
        );
```